docs/cse/schema/index.md

```diff
@@ -6,11 +6,73 @@ description: Learn about CSE Schema v3, schema attributes, and the Record proces
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<img src={useBaseUrl('img/icons/operations/schema.png')} alt="icon" width="75"/>
+This guide has information about Cloud SIEM Enterprise (CSE) schemas. In this section, we'll introduce the following concepts:
 
-import DocCardList from '@theme/DocCardList';
-import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
-
-In this section, we'll introduce the following concepts:
-
-<DocCardList items={useCurrentSidebarCategory().items}/>
+<div className="box-wrapper" markdown="1">
+<div className="box smallbox1 card">
```

Comment:
@kimsauce, could you flip box 1 and 2? "Record Processing Pipeline" should come first, and "CSE Schema Attributes" should come second.